### ISSUE TITLE
ExtTransferSwitch Not changing current limit 

### ISSUE THREAD

**Issue by Lowbank** [kwindrem/ExtTransferSwitch#4]
Hi Kevin I have installed your ExtTransferSwitch Package thank you. but still having problems. In the Device list the external AC input Transfer switch will change state form On grid to Generator with input change but In the GUI I see no change in the max current and not change in the heading shore power or generator when changing the input state.  

I have followed your instructions as follows.

While On grid, set the grid input current limit (in device list / MultiPlus-11/input current limit) and grid type (grid or shore).  (in setting/system setup/AC input 1)
Cause the transfer switch to change state and while On Generator, set the grid input current limit again.
Now the input current limit will switch between the grid and generator settings.  ( but this not happening )

Am I changing the current and grid type in the right spot?
Cheers Jon

**Comment by kwindrem** [comment]
Yes you are setting the current correctly. You should not need to change the input type when switching to generator. If that's not happening, the service probably isn't running.

The input type (grid/shore vs generator) should change when toggling the digital input. Also two separate current limits and two don't start generator when on AC 1 states are stored and will switch between them.

The item shown in the device list is simply the digital input which would change state even if the service that goes along with ExtTransferSwitch isn't running.

Check to see if there's a process running with that name. If not check the log: /data/log/ExtTransferSwitch/current.

You might try uninstalling then reinstalling ExtTrasnsferSwitch.

**Comment by Lowbank** [comment]
Thanks for your help. It will be a while till I go back on site. I have used the Multi Input current limit control Assistant for now. 
Can you point me to instructions on how to see if processes are running.

**Comment by kwindrem** [comment]
use ps

You get a lot of info so I typically filter it with grep

ps | grep Ext

would restrict the output to anything with "Ext" in the line

The log should show repeated starts if it's crashing.
